packages/api/endpoints/collections.js

```diff
@@ -69,6 +74,9 @@ async function post(req, res) {
     } catch (e) {
       if (e instanceof RecordDoesNotExist) {
         await c.create(data);
+        req.collectionRecord = data;
```

Comment:
This is a bit nitpicky and not strictly necessary, but really, we only need to add this data to the `req` object when in test mode, correct? If so, I'd prefer to nest these statements inside the `if(inTestMode())`. 

Also, to reduce some duplication, we could do

```
const response = { message: 'Record saved', record: data }.
if (inTestMode()) {
  req.returnMessage = response;
  return next();
}
return res.send(response);
```